docs/usage/help.rst.inc

```diff
@@ -8,16 +8,18 @@ borg help patterns
 
 File patterns support these styles: fnmatch, shell, regular expressions,
 path prefixes and path full-matches. By default, fnmatch is used for
-`--exclude` patterns and shell-style is used for `--pattern`. If followed
-by a colon (':') the first two characters of a pattern are used as a
+`--exclude` patterns and shell-style is used for the experimental `--pattern`
+option.
+
+If followed by a colon (':') the first two characters of a pattern are used as a
 style selector. Explicit style selection is necessary when a
 non-default style is desired or when the desired pattern starts with
 two alphanumeric characters followed by a colon (i.e. `aa:something/*`).
 
 `Fnmatch <https://docs.python.org/3/library/fnmatch.html>`_, selector `fm:`
 
     This is the default style for --exclude and --exclude-from.
-    These patterns use a variant of shell pattern syntax, with '*' matching
+    These patterns use a variant of shell pattern syntax, with '\*' matching
```

Comment:
btw. I don't think rST is _per the spec_ something that can be parsed efficiently without bloppers / hacks.